model/atmosphere/dycore/tests/dycore_stencil_tests/test_solve_tridiagonal_matrix_for_w_back_substitution.py

```diff
@@ -34,7 +42,7 @@ def reference(grid, z_q: np.array, w: np.array, **kwargs) -> dict:
         return dict(w=w_new)
 
     @pytest.fixture
-    def input_data(self, grid):
+    def input_data(self, grid: base.BaseGrid) -> dict:
```

Comment:
Same as above, shouldn't the return type of the `input_data` method return something like `dict[str, Field | scalar]`?

Reply:
cscs-ci run default